pkg/exchange/okex/query_order_trades_test.go

```diff
@@ -0,0 +1,46 @@
+package okex
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_QueryOrderTrades(t *testing.T) {
+	key := os.Getenv("OKEX_API_KEY")
+	secret := os.Getenv("OKEX_API_SECRET")
+	passphrase := os.Getenv("OKEX_API_PASSPHRASE")
+	if len(key) == 0 && len(secret) == 0 {
+		t.Skip("api key/secret are not configured")
+		return
+	}
+	if len(passphrase) == 0 {
+		t.Skip("passphrase are not configured")
+		return
+	}
```

Comment:
why not use testutil.IntegrationTestWithPassphraseConfigured here? I think you added that function?